.github/workflows/main.yml

```diff
@@ -82,12 +82,11 @@ jobs:
         with:
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
           onlyChanged: "!startsWith(github.ref, 'refs/heads/main')" # Use TurboSnap for PR builds
-          workingDir: ./packages/nimble-components
           externals: |
-            - '.storybook/public/**'
-            - '../packages/nimble-tokens/dist/icons/svg/**'
-            - '../packages/nimble-tokens/source/styledictionary/properties/**'
-          storybookBuildDir: ../../packages/site/dist/storybook
+            - './packages/nimble-components/.storybook/public/**'
+            - './packages/nimble-tokens/dist/icons/svg/**'
+            - './packages/nimble-tokens/source/styledictionary/properties/**'
+          storybookBuildDir: ./packages/site/dist/storybook
           storybookBaseDir: ./packages/nimble-components
```

Comment:
@m-akinc @fredvisser Tried removing thee workingDir config option and making the paths all from repo root. Seems to work, the PR description links to two build results, one simulating a branch doing an incremental change with snapshotting enabled and one simulating a change on main with snapshotting disabled.

Reply:
I'm gonna merge it to try it out, pretty minor change if we end up needing to tweak it